wiki/Manual.md

```diff
@@ -734,30 +734,27 @@ Each currency is an associative array (aka dictionary) with the following keys:
-    'base':    'BTC',         // uppercase string, unified base currency code, 3 or more letters
-    'quote':   'USD',         // uppercase string, unified quote currency code, 3 or more letters
     'baseId':  'btc',         // any string, exchange-specific base currency id
     'quoteId': 'usd',         // any string, exchange-specific quote currency id
-    'active':   true,         // boolean, market status
-    'taker':    0.002,        // taker fee rate, 0.002 = 0.2%
-    'maker':    0.0016,       // maker fee rate, 0.0016 = 0.16%
+    'settle':   'USDT'        // the unified currency code that the contract will settle in, only set if `contract` is true
+    'settleId': 'usdt'        // the currencyId of that the contract will settle in, only set if `contract` is true
     'type':    'spot'         // spot for spot, future for expiry futures, swap for perpetual swaps, 'option' for options
-    'linear':   true          // the contract is a linear contract (settled in quote currency)
-    'inverse':  false         // the contract is an inverse contract (settled in base currency)
-    'contractSize': 1         // the size of one contract, only used if `contract` is true
     'spot':     true,         // whether the market is a spot market
     'margin':   true,         // whether the market is a margin market
-    'future':   false,        // whether the market is a expiring future
     'swap':     false,        // whether the market is a perpetual swap
+    'future':   false,        // whether the market is a expiring future
     'option':   false,        // whether the market is an option contract
+    'active':   true,         // boolean, market status
     'contract': false,        // whether the market is a future, a perpetual swap, or an option
-    'settleId': 'usdt'        // the currencyId of that the contract will settle in, only set if `contract` is true
-    'settle':   'USDT'        // the unified currency code that the contract will settle in, only set if `contract` is true
+    'taker':    0.002,        // taker fee rate, 0.002 = 0.2%
+    'maker':    0.0016,       // maker fee rate, 0.0016 = 0.16%
+    'linear':   true          // the contract is a linear contract (settled in quote currency)
+    'inverse':  false         // the contract is an inverse contract (settled in base currency)
+    'contractSize': 1         // the size of one contract, only used if `contract` is true
     'expiry':  1641370465121  // the unix expiry timestamp in milliseconds, undefined for everything except market['type'] `future`
     'expiryDatetime': '2022-03-26T00:00:00.000Z' // The datetime contract will in iso8601 format
     'percentage': true,       // whether the taker and maker fee rate is a multiplier or a fixed flat amount
     'tierBased': false,       // whether the fee depends on your trading tier (your trading volume)
-    'feeSide': 'get'          // string literal can be 'get', 'give', 'base', 'quote', 'other'
```

Comment:
Let's keep the `feeSide` as well.

Reply:
In general, `taker`, `maker`, `percentage`, `tierBased` and `feeSide` should be grouped together, since those are fee-properties of a market.